README.md

```diff
@@ -480,3 +480,61 @@ In addition to the fields described in the BaseResponse, it has the following fi
 
 ---
 
+
+#### DapiApp.createACHTransfer
+
+Method is used to initiate a new ACH transfer.
+
+##### Method Description
+
+```c#
+public CreateACHTransferResponse createACHTransfer(Payment.ACHTransfer transfer, string accessToken, string userSecret)
+
+```
+
+##### Input Parameters
+
+| Parameter | Type | Description |
+|---|---|---|
+| **transfer** <br> _REQUIRED_ | `Payment.Transfer` | An object that contains info about the transfer that should be initiated. |
+| **accessToken** <br> _REQUIRED_ | `string` | Access Token obtained using the `exchangeToken` method. |
+| **userSecret** <br> _REQUIRED_ | `string` | The `userSecret` from a user’s successful log in to **Connect**. |
+
+##### Response
+
+In addition to the fields described in the BaseResponse, it has the following fields, which will only be returned if the status is `done`:
+
+| Parameter | Type | Description |
+|---|---|---|
+| reference | `string` | Transaction reference string returned by the bank. |
+
+---
+
+
+#### DapiApp.getACHPull
+
+Method is used to initiate a new ACH pull.
+
+##### Method Description
+
+```c#
+CGetACHPullResponse getACHPull(string accessToken, string userSecret, string operationID, UserInput[] userInputs)
+
+```
+
+##### Input Parameters
+
+| Parameter | Type | Description |
+|---|---|---|
+| **accessToken** <br> _REQUIRED_ | `string` | Access Token obtained using the `exchangeToken` method. |
+| **userSecret** <br> _REQUIRED_ | `string` | The `userSecret` from a user’s successful log in to **Connect**. |
+
```

Comment:
operationID field is required to locate the ACH transfer

Reply:
Also include the optional userInputs field in the description

Reply:
d37ce39dc5e43f6401602714948ba0bb2036b7f9